cmd/admin-handlers.go

```diff
@@ -182,7 +183,7 @@ func (a adminAPIHandlers) ServiceActionHandler(w http.ResponseWriter, r *http.Re
 	vars := mux.Vars(r)
 	action := vars["action"]
 
-	objectAPI := validateAdminReq(ctx, w, r)
+	objectAPI, _ := validateAdminReq(ctx, w, r, iampolicy.ServiceStartStopAdminAction)
```

Comment:
Only the root admin should have this permission

Reply:
If the root admin grants this permission to another admin, it should be allowed

Reply:
Can there be a scenario where an admin user with all the privileges as the root admin can exist?

Reply:
It's almost like full self control, but I guess serviceStop might be a bit of a stretch for admin users.

Perhaps we cannot allow it for now? And decide this later. We can always add this as a feature later.